tests/data/configs/hawq/resnet18_cifar10_mixed_int.json

```diff
@@ -20,10 +19,6 @@
                     "type": "hawq",
                     "num_data_points": 2,
                     "iter_number": 1
-                },
-                "batchnorm_adaptation": {
-                    "num_bn_adaptation_samples": 2,
-                    "num_bn_forget_samples": 1
```

Comment:
What's the reason for this, again? The default BN adaptation parameters will run ~1000 samples, which is probably too much for purposes of this test.

Reply:
First, this code was deleted and used in the sanity test only.
Second, the sanity test should check the whole pipeline, AutoQ calls BN adaptation multiple times. We should make sure e2e pipeline is OK for the user, it doesn't cause any multiprocessing issue or issue with multiple switches of the training mode during BN adaptation and etc.